### ISSUE TITLE
Failed while: Wait for Condition: InitialRolesPopulated: True when creating a cluster with a Standard User

### ISSUE THREAD

**Issue by izaac** [rancher/rancher#25744]
**What kind of request is this (question/bug/enhancement/feature request):**

Bug

**Steps to reproduce (least amount of steps as possible):**

- Install Rancher `2.4 master-head (03/03/2020)` commit id: 3623201ad
- Login as admin, create an Standard User (`standarduser1`)
- Logout and login as `standarduser1`
- Start creating a cluster, I used a one/all roles node on DigitalOcean
- Click Create

**Result:**

`Failed while: Wait for Condition: InitialRolesPopulated: True`

<img width="1119" alt="Screen Shot 2020-03-03 at 2 00 18 PM" src="https://user-images.githubusercontent.com/27659/75820355-6e2ae980-5d59-11ea-8c87-c718f4b4f1a8.png">

**Other details that may be helpful:**

- This is not reproducible in Rancher v2.3.5
- Clicking in Create again after getting the error succeeds in creating the cluster.

**Environment information**
- Rancher version: `2.4 master-head (03/03/2020)` commit id: 3623201ad
- Installation option (single install/HA): single

**Comment by izaac** [comment]
Rancher version: `2.4 master-head (03/05/2020)` commit id: f6872eb2c

Validated this is no longer happening when deploying a cluster with an Standard User.

**Comment by JannikZed** [comment]
I'm facing the same issue in Rancher 2.5.3 - saving it again does not succeed.. I'm an admin user.

**Comment by JannikZed** [comment]
..and for Rancher 2.6.3 as well. I'm trying to create a new cluster with the custom "hetzner driver"

**Comment by amagura** [comment]
...and for Rancher 2.5.8 as well.

**Comment by vullankib** [comment]
Do we know why this is happening and any resolution to this? Not sure why this is Marked as closed.

**Comment by ArthurMcTool** [comment]
Happening for me in 2.5.7. Would be nice to add a new cluster...

**Comment by ArthurMcTool** [comment]
I was getting this error in cattle-system pods:
`[rancher-7c7995df75-jwhq5] 2022/04/18 15:39:44 [ERROR] error syncing 'c-q89qh': handler mgmt-cluster-rbac-delete: Internal error occurred: failed calling webhook "rancherauth.cattle.io": Post "https://rancher-webhook.cattle-system.svc:443/v1/webhook/validation?timeout=10s": x509: certificate has expired or is not yet valid: current time 2022-04-18T15:39:44Z is after 2022-04-07T13:37:43Z, requeuing
`

This doc fixed me up:
https://rancher.com/docs/rancher/v2.6/en/troubleshooting/expired-webhook-certificates/

**Comment by skytodmoon** [comment]
，，，and for 2.7.1 as well

**Comment by gchait** [comment]
happens in 2.7.5 as well

**Comment by stvleung** [comment]
I spent a lot of time looking into the solution for this problem for us after we did an upgrade for Rancher. I ended starting a brand new Rancher installation and compared settings and stumbled on the the `rke2` feature flag. Once that is enabled (in the UI via "Global Settings -> Feature Flags"), then I was able to import clusters without issue. I still haven't found any documentation indicating that this would have lead to this error: `InitialRolesPopulated`.
